ConcernsCaseWork/ConcernsCaseWork/Pages/Trust/Overview.cshtml

```diff
@@ -40,22 +36,72 @@ else
 		<partial name="_TrustOverview" model="@Model.TrustDetailsModel"/>
 		<partial name="_Academies" model="@Model.TrustDetailsModel.Establishments"/>
 
-		<table class="govuk-table  govuk-!-margin-top-9">
-			<caption class="govuk-table__caption govuk-table__caption--m">
-				<div class="govuk-button-group flexalign__right">
-					<a asp-page="../case/concern/index" role="button" class="govuk-button govuk-button--secondary" data-module="govuk-button">
-						Create case
-					</a>
-				</div>
-				Active cases
-			</caption>
-			<partial name="_TrustActiveCases" model="@Model.ActiveCases"/>
-		</table>
-        <table class="govuk-table  govuk-!-margin-top-9">
-            <caption class="govuk-table__caption govuk-table__caption--m">
-                Closed cases
-            </caption>
-			<partial name="_TrustClosedCases" model="@Model.ClosedCases"/>
-		</table>
+		<nav class="moj-sub-navigation" aria-label="Sub navigation">
+        
+          <ul class="moj-sub-navigation__list">
+            <li class="moj-sub-navigation__item">
+	            <a id="hide-show-active" class="moj-sub-navigation__link" aria-expanded="true" aria-current="page" href="#active-cases"><span aria-hidden="false"></span>Active Cases</a>
+            </li>
+        
+            <li class="moj-sub-navigation__item">
+              <a id="hide-show-closed" class="moj-sub-navigation__link" href="#closed-cases" aria-expanded="false"><span aria-hidden="true"></span>Closed Cases</a>
+            </li>
+        
+          
+          </ul>
+        
+        </nav>
+		
+		<div id ="active-cases">
+			<table class="govuk-table   govuk-!-margin-top-9" data-module="moj-sortable-table">
+            			<caption class="govuk-table__caption govuk-table__caption--m">
+            				<div class="govuk-button-group flexalign__right">
+            					<a asp-page="../case/concern/index" role="button" class="govuk-button govuk-button--secondary" data-module="govuk-button">
+            						Create case
+            					</a>
+            				</div>
+            				Active cases
+            			</caption>
+            			<partial name="_TrustActiveCases" model="@Model.ActiveCases"/>
+            		</table>
+		</div>
+		<div id="closed-cases" hidden >
+			<table class="govuk-table  govuk-!-margin-top-9" data-module="moj-sortable-table">
+                    <caption class="govuk-table__caption govuk-table__caption--m">
+                        Closed cases
+                    </caption>
+            		<partial name="_TrustClosedCases" model="@Model.ClosedCases"/>
+            </table>
+		</div>
+		
+		
+		
 	</div>
-}
\ No newline at end of file
+
+
+<script type="application/javascript" nonce="@nonce">
```

Comment:
I think we also need these changes for the case management page

Case\Management\Index.cshtml